test/integration/pkg/client/resmgmt/resmgmt_queries_pvt_test.go

```diff
@@ -27,41 +28,44 @@ const (
 )
 
 func TestQueryCollectionsConfig(t *testing.T) {
-	sdk := mainSDK
+	//TODO: QueryCollectionsConfig for lifycycle
+	if metadata.Ccmode == "Lscc" {
```

Comment:
I think you can do the following to make the code more readable:
`if metadata.Ccmode != "Lscc" {
  t.Skip("this test is only valid for legacy chaincode")
}
`
Same goes for the other legacy tests.